src/System.Private.CoreLib/shared/System/Globalization/CultureData.Windows.cs

```diff
@@ -284,11 +287,11 @@ private string GetLanguageDisplayName(string cultureName)
 #else
             // Usually the UI culture shouldn't be different than what we got from WinRT except
             // if DefaultThreadCurrentUICulture was set
-            CultureInfo ci;
+            CultureInfo? ci;
 
             if (CultureInfo.DefaultThreadCurrentUICulture != null &&
                 ((ci = GetUserDefaultCulture()) != null) &&
-                !CultureInfo.DefaultThreadCurrentUICulture.Name.Equals(ci.Name))
+                !CultureInfo.DefaultThreadCurrentUICulture!.Name.Equals(ci!.Name))
```

Comment:
Why is the `!` on `CultureInfo.DefaultThreadCurrentUICulture` needed?  We just checked with a guard clause as part of the same `if`.
cc: @jaredpar

Reply:
i think this is fixed with new compiler version (per Santi)

Reply:
Yep, did you tried it yet with the new compiler we pulled in?

Reply:
haven't built on Windows yet